arroyo-console/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import {
+  Button,
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react';
+import { TourContext, TourSteps } from '../tour';
+
+const WelcomeModal: React.FC = () => {
+  const { tourStep, setTourStep, disableTour } = useContext(TourContext);
+
+  return (
+    <Modal
+      isOpen={tourStep == TourSteps.WelcomeModal}
+      onClose={() => {}}
+      isCentered
+      size={'xl'}
+      variant={'tour'}
+    >
+      <ModalOverlay />
+      <ModalContent padding={2}>
+        <ModalCloseButton onClick={disableTour} />
+        <ModalBody display={'flex'} gap={3} flexDirection={'column'} alignItems={'center'}>
+          <img width="320" src="/logo.svg" />
+          <Text fontSize={'lg'} fontWeight={'bold'} textAlign={'center'}>
+            Welcome!
+          </Text>
+          <Text fontSize={'md'} textAlign={'center'}>
+            Arroyo is distributed stream processing engine, designed to efficiently perform stateful
+            computations on streams of data.
+            <br />
+            <br />
+            Want some help running your first pipeline?
+          </Text>
+        </ModalBody>
+        <ModalFooter
+          display={'flex'}
+          flexDirection={'column'}
+          gap={'5px'}
+          justifyContent={'center'}
+        >
+          <Button colorScheme={'blue'} onClick={() => setTourStep(TourSteps.CreatePipelineButton)}>
+            Let's do it!
+          </Button>
+          <Link>
+            <Text fontSize={'sm'} color={'grey.200'} onClick={disableTour}>
+              Maybe later
```

Comment:
If you click this, is there a way for the user to take the tour later? Otherwise maybe we want to change the text.

Reply:
Currently there isn't, so I changed the text to "No thanks". Or we could a button at the bottom of the sidebar to start the tour?